worldguard-legacy/src/main/java/com/sk89q/worldguard/bukkit/listener/WorldGuardBlockListener.java

```diff
@@ -509,6 +509,13 @@ public void onBlockForm(BlockFormEvent event) {
                     return;
                 }
             }
+
+            if (type == BlockID.FROSTED_ICE && wcfg.useRegions &&
+                    !plugin.getGlobalRegionManager().allows(
+                            DefaultFlag.FROSTED_ICE_FORM, event.getBlock().getLocation())) {
+                event.setCancelled(true);
+                return;
```

Comment:
I can remove it if that's desired. Added it as it was present in the same way a lot of other places.